Source/Bogus/DataSets/System.cs

```diff
@@ -17,41 +17,47 @@ public class System : DataSet
    /// <param name="locale">The locale that will be used to generate values.</param>
    public System(string locale = "en") : base(locale)
    {
-      mimes = this.GetArray("mimeTypes");
-
-      lookup = mimes.OfType<BObject>()
-         .ToDictionary(o => o["mime"].StringValue);
-
-      mimeKeys = mimes
-         .OfType<BObject>()
-         .Select(o => o["mime"].StringValue)
-         .Distinct()
-         .ToArray();
-
-      exts = mimes
-         .OfType<BObject>()
-         .SelectMany(bObject =>
+      mimes = new Lazy<BArray>(
+         () => this.GetArray("mimeTypes"));
+
+      lookup = new Lazy<Dictionary<string, BObject>>(
+         () => mimes.Value.OfType<BObject>()
+            .ToDictionary(o => o["mime"].StringValue));
+      
+      mimeKeys = new Lazy<string[]>(
+         () => mimes.Value
+            .OfType<BObject>()
+            .Select(o => o["mime"].StringValue)
+            .Distinct()
+            .ToArray());
+      
+      exts = new Lazy<string[]>(
+         () => mimes.Value
+            .OfType<BObject>()
+            .SelectMany(bObject =>
             {
-               if( bObject.ContainsKey("extensions") )
```

Comment:
also, if you could; small nit: avoid these small reformatting changes in existing source that doesn't change.